packages/appium/lib/constants.js

```diff
@@ -40,13 +41,8 @@ export const KNOWN_DRIVERS = Object.freeze(
   /** @type {const} */ ({
     uiautomator2: 'appium-uiautomator2-driver',
     xcuitest: 'appium-xcuitest-driver',
-    youiengine: 'appium-youiengine-driver',
-    windows: 'appium-windows-driver',
```

Comment:
we actually officially support the driver itself, just not the server part of it. It's complicated %)

Reply:
i'm fine to put it back, but how optimistic are we that this support will continue to make sense? the "driver" from a user perspective is not just the node.js part but the whole package. what are we trying to say about that?

Reply:
I don't say anything. I assume M$ should ;)
According the current state of things we can remove it